src/citrine/_serialization/properties.py

```diff
@@ -738,12 +852,9 @@ def _serialize(self, value: typing.Any) -> dict:
             return value.dump()
 
     def _deserialize(self, value: dict):
-        if 'type' in value and value['type'] == LinkByUID.typ:
-            if 'scope' in value and 'id' in value:
-                value.pop('type')
-                return LinkByUID(**value)
-            else:
-                raise ValueError("LinkByUID dictionary must have both scope and id fields")
+        if 'type' in value:
+            target = DictSerializable.class_mapping[value['type']]
+            return target.build(value)
```

Comment:
No need for LinkOrElse to know how to decode a LinkByUID.

Reply:
Why not?

Reply:
Because it already needs to know to ask `DictSerializable` to decode the object, and `DictSerializable` knows how to decode LinkByUIDs.